### PR TITLE
62 restore data table

### LINKED ISSUE #62: Restore data table
Let's get the data table working again.

- User should be able to use the buttons in the header to toggle between map and table view. (Header needs to be brought into `App` component)
- Now may be the time to consider a different table that may play better with react and be more performant.

### PR DESCRIPTION
**Pull request by chriswhong**
- Implements `react-table` to create a paginated table view to complement the map view.
- Moves navbar into `App`, so view button clicks can be handled
- Creates a top-level state property `rows`, which is used by the new `Table` component. 
- Refactors `/tiles/initialize` endpoint to return `rows` instead of `featureCount` (featureCount is calculated in the client from rows.length)
- For geojson mode, rows is calculated from the FeatureCollection once it is received in the client.

- Cleans up unused dependencies

Closes #62

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,2639 +0,0 @@
-{
-  "name": "postgis-preview",
```
Comment (brianbancroft) [inline package-lock.json:2]: Why are we ditching package-lock?

File: public/js/script.js [hunk 2]
```diff
@@ -1,260 +0,0 @@
-(function init() {
```
Comment (brianbancroft) [inline public/js/script.js:1]: 🎉

**Review by brianbancroft — APPROVED** [review]
So this works in all my attempts, and it doesn't crash. I'd merge this as-is. But if you're up for it, there are a couple things that might be worth crunching before the merge:

1. I've always been taught that we should always commit [package-lock.json](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5). I'm not aware of a persuasive argument otherwise. 
2. The text that appears when there's no rows selected. It might merit a few extra words explaining what's going on, a few points larger on the typeface, and possibly a margin/padding at the top and left?

image

File: package-lock.json (hunk 1 above)
Comment (chriswhong) [reply package-lock.json:2]: We need to go with yarn or npm, our team's been going the yarn route lately.  Do you have a preference?

**Comment by chriswhong** [comment]
1) yarn.lock is committed, we shouldn't commit both `yarn.lock` and `package-lock.json`
2) I think both the map and the table view should be grayed out or something if there is no result being displayed... we should make an issue for this, but I'm fine leaving it as-is for the time being.

**Comment by brianbancroft** [comment]
Both good explanations. Merging!

**Pull request merged**